db/schema.rb

```diff
@@ -178,9 +178,6 @@
     t.index ["uuid"], name: "index_courses_on_uuid", unique: true
   end
 
-  create_table "data_migrations", primary_key: "version", id: :string, force: :cascade do |t|
```

Comment:
We decided we were going to keep this in main. It probably got removed as you don't have it on your local database.